modules/database/test/std/rec/Makefile

```diff
@@ -93,6 +93,13 @@ testHarness_SRCS += longoutTest.c
 TESTFILES += ../longoutTest.db
 TESTS += longoutTest
 
+TESTPROD_HOST += boTest
+boTest_SRCS += boTest.c
+boTest_SRCS += recTestIoc_registerRecordDeviceDriver.cpp
+testHarness_SRCS += boTest.c
```

Comment:
Adding to the testHarness executable also requires a change to `modules/database/test/std/rec/epicsRunRecordTests.c`.  eg. the corresponding change to hook in `MAIN(analogMonitorTest)`.

https://github.com/epics-base/epics-base/blob/fa00572780a00bea2f57dbff69653b14bd792be0/modules/database/test/std/rec/epicsRunRecordTests.c#L16

https://github.com/epics-base/epics-base/blob/fa00572780a00bea2f57dbff69653b14bd792be0/modules/database/test/std/rec/epicsRunRecordTests.c#L32

Reply:
Resolved in https://github.com/epics-base/epics-base/pull/340/commits/6e8955f93ef14dbeda8596a82b87c876de6c8587